### ISSUE TITLE
[bug] TAS buttons continue to be pressed while the game is paused

### ISSUE THREAD

**Issue by ThisAMJ** [p2sr/SourceAutoRecord#54]
### Describe the bug

When playing a TAS, if you are to pause the game the `tastick` hud continues to tick up and certain tools (namely buttons field, unsure of others) continue to play.

### Steps to reproduce

Create a TAS with the following script
```p2tas
start map sp_a1_intro4
// Smooth jazz for fun
200>
repeat 100
+100>||B1
end
```
and play it. If you pause and unpause, the next blue portal shot will fire sooner than it should.

**Comment by RainbowwPhoenixx** [comment]
So I was wrong and  it doesn't only affect buttons, after doing some experiments it turns out the TAS just keeps running during pauses. I have managed to fix this in a specific case but it often desyncs. I don't think it is possible to prevent these desyncs simply because pause abuse is a thing.

**Comment by mlugg** [comment]
Addressed by #50 in 8fcc5ee, but might be able to be made more consistent
